packages/cdc-react/src/components/Card/Card.tsx

```diff
@@ -17,42 +17,57 @@ interface CardProps {
   actionButtonOnClick?: MouseEventHandler<HTMLDivElement> &
     MouseEventHandler<HTMLButtonElement>;
   children: React.ReactNode;
+  className: string;
```

Comment:
If you're using the `JSX.IntrinsicElements` in the props definition, you shouldn't need to add it to the Card specific props here.

Reply:
good catch, removing